code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -395,8 +395,8 @@
  * * name for debugging purposes
  * Other vars such as alpha will automatically be applied with the render source
  */
-/atom/movable/screen/plane_master/proc/generate_render_relays()
-	var/relay_loc = "CENTER"
+/atom/movable/screen/plane_master/proc/generate_render_relays(relay_loc_override)
+	var/relay_loc = relay_loc_override ? relay_loc_override : "CENTER"
```

Comment:
rather then doin it like this, we should just read off home. no need for a default either

Reply:
ok i hope that's what you mean